app/views/requirements/_form.html.erb

```diff
@@ -16,9 +16,9 @@
       <%# render 'requirement_step_fields', :f => reqstep %>
```

Comment:
I feel like this whole set of lines should probably just be deleted, unless we think we'll be re-adding multi-step requirements (seems unlikely to me...).